app/schemas/models/level.coffee

```diff
@@ -211,7 +211,17 @@ IntroContentObject = {
   additionalProperties: false,
   properties: {
     type: { title: 'Content type', enum: ['cinematic', 'interactive', 'cutscene-video', 'avatarSelectionScreen'] }
-    contentId: c.stringID(title: 'Content ID')
+    contentId: {
+      oneOf: [
+        c.stringID(title: 'Content ID for all languages')
+        {
+          type: 'object',
+          title: 'Content ID specific to languages',
+          additionalProperties: c.stringID(),
```

Comment:
Is this `additionalProperties` the value for the language key?
Would an example of this object look like this?

```
{
  javascript: <stringID here>
}
```

Reply:
That is correct!